src/hooks/useConnectOwnedVenues.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { createContext, useContext, useMemo, useCallback } from "react";
+
+import { SparkleSelector } from "types/SparkleSelector";
+import { ReactHook } from "types/utility";
+import { AnyVenue } from "types/venues";
+
+import { WithId, withId } from "utils/id";
+
+import { useFirestoreConnect, isLoaded } from "hooks/useFirestoreConnect";
+import { useSelector } from "hooks/useSelector";
+
+export const ownedVenuesSelector: SparkleSelector<
+  Record<string, AnyVenue> | undefined
+> = (state) => state.firestore.data.ownedVenues;
```

Comment:
move into the selectors

Reply:
moved 8819037c8d4dc531a31279aa15a3e1979048a188